index.js

```diff
@@ -34,6 +37,17 @@ var
     return whitelisted;
   },
 
+  add = function add(collection, records, cb) {
+    var newCollection = collection.concat(records);
+
+    if (cb) {
+      cb(null, newCollection);
```

Comment:
I may have overlooked that. I'll go over it again and update the code. Thanks :+1: